docs/conf.py

```diff
@@ -57,6 +58,9 @@
 version = VERSION
 # The full version, including alpha/beta/rc tags.
 release = VERSION
+# Try to find out the hash of the version.
+repo = git.Repo("..")
+hash_version = str(repo.head.commit.hexsha[:7])
```

Comment:
I rewrite the stuff with the GitPython module. Dunno if it works on readthedoc or not.

Reply:
Readthedocs is fine if we tell him to use the `test-requirements.txt` file, otherwise `GitPython` isn't installed.

Reply:
Could you, please, elaborate a little bit on your comment. Do you prefer me to put GitPython in `requierments.txt`?

Reply:
no using `test-requirements.txt` is perfectly fine :)

Reply:
👍